src/cbuild/hooks/post_install/007_collect_debug_sources.py

```diff
@@ -0,0 +1,64 @@
+from cbuild.core import chroot
+
+import pathlib
+import shutil
+
+
+def _scan_file(pkg, v, processed):
+    # silently ignore invalid files
+    for source_file in (
+        chroot.enter(
+            "list-dwarf-srcfiles",
+            f"/{v.relative_to(pkg.rparent.bldroot_path)}",
+            capture_output=True,
+            ro_root=True,
+            ro_build=True,
+            unshare_all=True,
+        )
+        .stdout.strip()
+        .splitlines()
+    ):
+        source_file = pathlib.Path(source_file.strip().decode())
+
+        # don't have access to these!
+        if source_file.name in ["<stdin>"]:
+            continue
+
```

Comment:
after removing the prefix changes:

```suggestion
        if source_file.name in ["<stdin>"]:
            continue

		# /usr sourcefiles like /usr/include headers are already provided by -devel packages
		if source_file.parent.is_relative_to("/usr"):
			continue
```